src/Directory.Packages.props

```diff
@@ -5,5 +5,6 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageVersion Include="Microsoft.Build" Version="17.13.0-dev-24518-01" />
+    <PackageVersion Include="DotUtils.MsBuild.BinlogRedactor.SensitiveDataDetector" Version="0.0.9" />
```

Comment:
Btw. this feels as a quite a version bump (0.0.4-beta -> 0.0.9)

Reply:
It's a different package, the previous version was 0.0.8
https://www.nuget.org/packages/DotUtils.MsBuild.BinlogRedactor.SensitiveDataDetector/

Reply:
🤦  :-D
Sry - I was just too quick going through this